src/features/surveys/components/surveyForm/SurveyFooter.tsx

```diff
@@ -0,0 +1,87 @@
+import { FC } from 'react';
+import { Box, List, ListItem, Typography } from '@mui/material';
+import Image from 'next/image';
+import Link from 'next/link';
+
+import messageIds from 'features/surveys/l10n/messageIds';
+import { Msg } from 'core/i18n';
+import SurveyContainer from './SurveyContainer';
+import theme from 'theme';
+
+const links = [
+  {
+    href: 'https://zetkin.org/',
+    text: messageIds.surveyFooter.links.foundation,
+  },
+  {
+    href: 'https://manual.zetkin.org/',
+    text: messageIds.surveyFooter.links.manual,
+  },
+  {
+    href: 'https://zetkin.org/privacy',
+    text: messageIds.surveyFooter.links.privacy,
+  },
```

Comment:
This should be using the environment variable (same as in `SurveyPrivacyPolicy`).

Reply:
✅ https://github.com/zetkin/app.zetkin.org/pull/2447/commits/213b7b382887e620afea05013e06a9c031275083